src/net.cpp

```diff
@@ -45,6 +45,8 @@
 
 // We add a random period time (0 to 1 seconds) to feeler connections to prevent synchronization.
 #define FEELER_SLEEP_WINDOW 1
+/** Run the feeler connection loop once every 2 minutes or 120 seconds. **/
+static const int FEELER_INTERVAL = 60;
```

Comment:
If we are going to move the FEELER_INTERVAL to net.cpp why not move the other static const int intervals such as PING_INTERVAL or TIMEOUT_INTERVAL in net.h as well?